jupyter_client/multikernelmanager.py

```diff
@@ -104,7 +104,6 @@ def start_kernel(self, kernel_name=None, **kwargs):
         km = self.kernel_manager_factory(connection_file=os.path.join(
                     self.connection_dir, "kernel-%s.json" % kernel_id),
                     parent=self, log=self.log, kernel_name=kernel_name,
-                    **kwargs
```

Comment:
Oh, now that I read it I see the problem. `kernel_spec_manager` should be passed to the constructor and kwargs should be passed on to start. You are right to remove kwargs from the constructor.